src/rapids_metadata/metadata.py

```diff
@@ -45,6 +45,11 @@ class RAPIDSPackage:
         description="""Whether or not the package has a CUDA suffix.""",
     )
 
+    is_conda_only: bool = Field(
+        default=False,
+        description="""Whether or not the package exists only as a conda package.""",
+    )
+
```

Comment:
Let's have separate fields for conda and wheels:

```suggestion
    has_conda_package: bool = Field(
        default=True,
        description="""Whether or not the package exists as a conda package.""",
    )

    has_wheel_package: bool = Field(
        default=True,
        description="""Whether or not the package exists as a wheel package.""",
    )
```

Then packages that don't produce wheels can set `has_wheel_package` to `False`.